pallets/automation-time/src/benchmarking.rs

```diff
@@ -354,9 +360,9 @@ benchmarks! {
 		for i in 0..v {
 			let provided_id: Vec<u8> = vec![i.saturated_into::<u8>()];
 			let task = TaskOf::<T>::create_event_task::<T>(caller.clone(), provided_id.clone(), vec![time.into()], vec![4, 5, 6]).unwrap();
-			let task_id = AutomationTime::<T>::schedule_task(&task, provided_id).unwrap();
-			let missed_task = MissedTaskV2Of::<T>::new(caller.clone(), task_id, time.into());
-			<AccountTasks<T>>::insert(caller.clone(), task_id, task);
+			let task_id = AutomationTime::<T>::schedule_task(&task).unwrap();
+			let missed_task = MissedTaskV2Of::<T>::new(caller.clone(), task_id.clone(), time.into());
+			<AccountTasks<T>>::insert(caller.clone(), task_id.clone(), task);
```

Comment:
I notice there are several addition of the `.clone()`. Is the `task_id.clone()` to create a deep clone to prevent the original variable from getting destroyed outside the insert() call?

Reply:
Ah I see. Okay, hmm yeah changing underlining signagure is too big of a change. I got it. Let’s keep them as-is.